src/components/PerfPlugin/TrendChartsPlugin.tsx

```diff
@@ -1,12 +1,12 @@
 import React from "react";
-import TrendCharts from "@mongodb-dev-prod/trend-charts-ui";
+import {getSignalProcessingUrl} from "../../utils/environmentalVariables";
 
 interface Props {
   taskId: string;
 }
 
 const TrendChartsPlugin: React.FC<Props> = ({ taskId }) => (
-  <TrendCharts taskId={taskId} />
+    <iframe style={{width:"100%", minHeight: "1000px"}} src={`${getSignalProcessingUrl()}/task/${taskId}/performanceData`} title="Task Performance Data"/>
```

Comment:
Can you use emotion to style this element. 

```
import styled from "@emotion/styled";

const StyledIframe = styled(iframe)`
width: 100%;
height: 1000px;
`
```

Reply:
Why 1000px for the height?

Reply:
I just set that min height since it was defaulting to something quite small like 50px. Should I just set height to 100% and let spruce handle the size of the wrapping component?